ic-os/guestos/envs/dev-malicious/BUILD.bazel

```diff
@@ -7,6 +7,7 @@ load("//ic-os/guestos:defs.bzl", "image_deps")
 #    or //ic-os/defs.bzl for the full list of targets.
 icos_build(
     name = "dev-malicious",
+    build_local_base_image = True,
```

Comment:
As far as I understand, this builds the base image from scratch instead of pulling it from the registry. Why is that necessary?

Reply:
While I've been testing on CI, I'm skipping over the registry at the loss of determinism, so that I'm not pushing a bunch of new images. When we're ready to roll, I'll need to break those changes to a separate PR to push the new image.

Reply:
The later, this builds the base image directly in the pipeline rather than looking for it. The [`container-base-images.yml`](https://github.com/dfinity/ic/actions/workflows/container-base-images.yml) workflow pushes these regularly.

I've switched back to a draft until we're ready to push the image 👍.